package.json

```diff
@@ -22,7 +22,9 @@
     "chai": "^2.2.0",
     "chai-as-promised": "^4.3.0",
     "coveralls": "^2.11.2",
-    "lab": "^5.5.1",
-    "nano": "^6.1.2"
+    "lab": "^5.5.1"
+  },
+  "dependencies": {
+    "nano": "^6.2.0"
```

Comment:
`nano` isn't technically a dependency since we inject it into `nano-promises`. I could be useful to have it as a peer dependency though. Could you move it back to devDependencies?